src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamProxy.cs

```diff
@@ -33,6 +33,12 @@ public async Task<IWebTokenRequestResultWrapper> GetTokenSilentlyAsync(WebAccoun
         {
             using (_logger.LogBlockDuration("WAM:GetTokenSilentlyAsync:webAccount"))
             {
+                if (_logger.IsLoggingEnabled(LogLevel.Verbose))
```

Comment:
should this be pii ?

Reply:
This is a very good point @jennyf19 , thanks for spotting it!